[2025-03-07T11:03:00.094+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.delete_data_from_table scheduled__2025-03-04T00:00:00+00:00 [queued]>
[2025-03-07T11:03:00.105+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.delete_data_from_table scheduled__2025-03-04T00:00:00+00:00 [queued]>
[2025-03-07T11:03:00.105+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2025-03-07T11:03:00.119+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): delete_data_from_table> on 2025-03-04 00:00:00+00:00
[2025-03-07T11:03:00.127+0000] {standard_task_runner.py:57} INFO - Started process 1037 to run task
[2025-03-07T11:03:00.131+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'delete_data_from_table', 'scheduled__2025-03-04T00:00:00+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/postgres_operator.py', '--cfg-path', '/tmp/tmpdawoy3l0']
[2025-03-07T11:03:00.134+0000] {standard_task_runner.py:85} INFO - Job 35: Subtask delete_data_from_table
[2025-03-07T11:03:00.184+0000] {task_command.py:416} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.delete_data_from_table scheduled__2025-03-04T00:00:00+00:00 [running]> on host 5588030d4cfa
[2025-03-07T11:03:00.269+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v03' AIRFLOW_CTX_TASK_ID='delete_data_from_table' AIRFLOW_CTX_EXECUTION_DATE='2025-03-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-04T00:00:00+00:00'
[2025-03-07T11:03:00.270+0000] {sql.py:274} INFO - Executing: 
            delete from dag_runs where dt = '2025-03-04' and dag_id = 'dag_with_postgres_operator_v03';
        
[2025-03-07T11:03:00.277+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-03-07T11:03:00.287+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-03-07T11:03:00.291+0000] {sql.py:418} INFO - Running statement: 
            delete from dag_runs where dt = '2025-03-04' and dag_id = 'dag_with_postgres_operator_v03';
        , parameters: None
[2025-03-07T11:03:00.293+0000] {sql.py:427} INFO - Rows affected: 0
[2025-03-07T11:03:00.303+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v03, task_id=delete_data_from_table, execution_date=20250304T000000, start_date=20250307T110300, end_date=20250307T110300
[2025-03-07T11:03:00.344+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-07T11:03:00.364+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-07T11:32:46.551+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.delete_data_from_table scheduled__2025-03-04T00:00:00+00:00 [queued]>
[2025-03-07T11:32:46.558+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.delete_data_from_table scheduled__2025-03-04T00:00:00+00:00 [queued]>
[2025-03-07T11:32:46.558+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2025-03-07T11:32:46.575+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): delete_data_from_table> on 2025-03-04 00:00:00+00:00
[2025-03-07T11:32:46.582+0000] {standard_task_runner.py:57} INFO - Started process 866 to run task
[2025-03-07T11:32:46.585+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'delete_data_from_table', 'scheduled__2025-03-04T00:00:00+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/postgres_operator.py', '--cfg-path', '/tmp/tmp_h6do1la']
[2025-03-07T11:32:46.587+0000] {standard_task_runner.py:85} INFO - Job 24: Subtask delete_data_from_table
[2025-03-07T11:32:46.627+0000] {task_command.py:416} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.delete_data_from_table scheduled__2025-03-04T00:00:00+00:00 [running]> on host 5837dac536a6
[2025-03-07T11:32:46.700+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v03' AIRFLOW_CTX_TASK_ID='delete_data_from_table' AIRFLOW_CTX_EXECUTION_DATE='2025-03-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-04T00:00:00+00:00'
[2025-03-07T11:32:46.700+0000] {sql.py:274} INFO - Executing: 
            delete from dag_runs where dt = '2025-03-04' and dag_id = 'dag_with_postgres_operator_v03';
        
[2025-03-07T11:32:46.710+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-03-07T11:32:46.718+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-03-07T11:32:46.721+0000] {sql.py:418} INFO - Running statement: 
            delete from dag_runs where dt = '2025-03-04' and dag_id = 'dag_with_postgres_operator_v03';
        , parameters: None
[2025-03-07T11:32:46.722+0000] {sql.py:427} INFO - Rows affected: 0
[2025-03-07T11:32:46.732+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v03, task_id=delete_data_from_table, execution_date=20250304T000000, start_date=20250307T113246, end_date=20250307T113246
[2025-03-07T11:32:46.757+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-07T11:32:46.772+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-07T12:05:06.787+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.delete_data_from_table scheduled__2025-03-04T00:00:00+00:00 [queued]>
[2025-03-07T12:05:06.794+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.delete_data_from_table scheduled__2025-03-04T00:00:00+00:00 [queued]>
[2025-03-07T12:05:06.794+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2025-03-07T12:05:06.806+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): delete_data_from_table> on 2025-03-04 00:00:00+00:00
[2025-03-07T12:05:06.810+0000] {standard_task_runner.py:57} INFO - Started process 481 to run task
[2025-03-07T12:05:06.812+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'delete_data_from_table', 'scheduled__2025-03-04T00:00:00+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/postgres_operator.py', '--cfg-path', '/tmp/tmp00akj7_8']
[2025-03-07T12:05:06.813+0000] {standard_task_runner.py:85} INFO - Job 12: Subtask delete_data_from_table
[2025-03-07T12:05:06.849+0000] {task_command.py:416} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.delete_data_from_table scheduled__2025-03-04T00:00:00+00:00 [running]> on host afc6f8faa09c
[2025-03-07T12:05:06.930+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v03' AIRFLOW_CTX_TASK_ID='delete_data_from_table' AIRFLOW_CTX_EXECUTION_DATE='2025-03-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-04T00:00:00+00:00'
[2025-03-07T12:05:06.931+0000] {sql.py:274} INFO - Executing: 
            delete from dag_runs where dt = '2025-03-04' and dag_id = 'dag_with_postgres_operator_v03';
        
[2025-03-07T12:05:06.942+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-03-07T12:05:06.954+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-03-07T12:05:06.959+0000] {sql.py:418} INFO - Running statement: 
            delete from dag_runs where dt = '2025-03-04' and dag_id = 'dag_with_postgres_operator_v03';
        , parameters: None
[2025-03-07T12:05:06.961+0000] {sql.py:427} INFO - Rows affected: 0
[2025-03-07T12:05:06.976+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v03, task_id=delete_data_from_table, execution_date=20250304T000000, start_date=20250307T120506, end_date=20250307T120506
[2025-03-07T12:05:07.025+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-07T12:05:07.054+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
